### PR TITLE
update(HTML): web/html/element/a

### PR DESCRIPTION
**Pull request by undead404**
Оригінальний вміст: ["&lt;a&gt;: Якірний елемент"@MDN](https://developer.mozilla.org/en-us/docs/Web/HTML/Element/a), [сирці "&lt;a&gt;: Якірний елемент"@GitHub](https://github.com/mdn/content/blob/main/files/en-us/web/html/element/a/index.md)

Нові зміни:
- [Convert noteblocks for web/html folder (#35114)](https://github.com/mdn/content/commit/9c09b183a5ce844a75c2f22e909d03f71ca329fc)
- [Convert noteblocks for web/html/element folder (#35085)](https://github.com/mdn/content/commit/b7955e77cd4293adf45ef23686df50b0305f02ad)
- [Add two common relative URL forms for context (#34874)](https://github.com/mdn/content/commit/6fc5ef907c62d992cc2d7227cc88af3da1efa1b9)
- [HTML element attr list should link to attr page (#34523)](https://github.com/mdn/content/commit/991385e7cfb9ac8589332b07aadcc4b38edea512)
- [Move Accessibility section above Examples for html element pages (#34007)](https://github.com/mdn/content/commit/bde0cb215d1d307c08678abe6623fc0d39f4cf7f)
- [feat: Editorial review: Attribution reporting API (#33172)](https://github.com/mdn/content/commit/ba5f28ab10ef1af48a55f363c8facc04a1f94479)

### CODE REVIEW

File: files/uk/web/html/element/a/index.md [hunk 1]
```diff
@@ -17,6 +17,27 @@ browser-compat: html.elements.a
 
 Атрибути цього елемента включають [глобальні атрибути](/uk/docs/Web/HTML/Global_attributes).
 
+- `attributionsrc` {{experimental_inline}}
+
+  - : Задає те, що ви хочете, аби браузер надіслав заголовок {{httpheader("Attribution-Reporting-Eligible")}}. З серверного боку це використовується для пуску надсилання заголовка {{httpheader("Attribution-Reporting-Register-Source")}} у відповіді, щоб зареєструвати [джерело атрибуції на основі навігації](/uk/docs/Web/API/Attribution_Reporting_API/Registering_sources#dzherela-atrybutsii-na-osnovi-navihatsii).
+
+    Браузер зберігає дані про джерело, пов'язані з джерелом атрибуцї, заснованим на навігації (таким, яке задано в заголовку відповіді {{httpheader("Attribution-Reporting-Register-Source")}}), коли користувач клацає посилання. Дивіться подробиці в [API звітування про атрибуцію](/uk/docs/Web/API/Attribution_Reporting_API).
```
[the remark this answers is not in the document]
Comment (undead404) [reply files/uk/web/html/element/a/index.md:24]:
```suggestion
    Браузер зберігає дані про джерело, пов'язані з джерелом атрибуції, заснованим на навігації (таким, яке задано в заголовку відповіді {{httpheader("Attribution-Reporting-Register-Source")}}), коли користувач клацає посилання. Дивіться подробиці в [API звітування про атрибуцію](/uk/docs/Web/API/Attribution_Reporting_API).
```

File: files/uk/web/html/element/a/index.md [hunk 2]
```diff
@@ -17,6 +17,28 @@ browser-compat: html.elements.a
...
+      attributionsrc="https://a.example/register-source
+      https://b.example/register-source"
+      ```
+
+      Це корисно в тих випадках, коли запитаний ресурс не перебуває на тому сервері, який ви контролюєте, або просто треба обробляти реєстрацію джерела атрибуції на іншому сервері. У такому випадку можна задати один або більше URL як значення `attributionsrc`. Коли стається запит ресурсу, на URL, задані в `attributionsrc`, надсилається заголовок {{httpheader("Attribution-Reporting-Eligible")}}, на додачу до походження ресурсу. Ці URL можуть потім відповісти заголовком {{httpheader("Attribution-Reporting-Register-Source")}}, щоб завершити реєстрацію.
+
+      > [!NOTE]
+      > Задання кількох URL означає те, що на одну можливість може бути зареєстровано більш ніж одне джерело атрибуції. Можна, наприклад, мати різні кампанії, успішність яких вимірюється, що включає генерування різних звітів щодо різних даних.
+      > Елементи `<a>` не можуть використовуватися як пускачі атрибуції, а лише як її джерела.
+
```
Comment (alinkedd) [inline files/uk/web/html/element/a/index.md:41]: тут те саме - передивись, як ми змержили, то так і пофікси, щоб було у всіх файлах однаково

**Review by alinkedd — CHANGES_REQUESTED** [review]

File: files/uk/web/html/element/a/index.md [hunk 3]
```diff
@@ -17,6 +17,28 @@ browser-compat: html.elements.a
 
 Атрибути цього елемента включають [глобальні атрибути](/uk/docs/Web/HTML/Global_attributes).
 
+- `attributionsrc` {{experimental_inline}}
+
+  - : Задає те, що ви хочете, аби браузер надіслав заголовок {{httpheader("Attribution-Reporting-Eligible")}}. З серверного боку це використовується для пуску надсилання заголовка {{httpheader("Attribution-Reporting-Register-Source")}} у відповіді, щоб зареєструвати [джерело атрибуції на основі навігації](/uk/docs/Web/API/Attribution_Reporting_API/Registering_sources#dzherela-atrybutsii-na-osnovi-navihatsii).
```
Comment (undead404) [inline files/uk/web/html/element/a/index.md:22]:
```suggestion
  - : Задає те, що ви хочете, аби браузер надіслав заголовок {{httpheader("Attribution-Reporting-Eligible")}}. На серверному боці такий заголовок використовується для запуску надсилання у відповіді заголовка {{httpheader("Attribution-Reporting-Register-Source")}} у відповіді, щоб зареєструвати [джерело атрибуції на основі навігації](/uk/docs/Web/API/Attribution_Reporting_API/Registering_sources#dzherela-atrybutsii-na-osnovi-navihatsii).
```

File: files/uk/web/html/element/a/index.md [hunk 4]
```diff
@@ -17,6 +17,28 @@ browser-compat: html.elements.a
...
+
+    Браузер зберігає дані про джерело, пов'язані з джерелом атрибуції, заснованим на навігації (таким, яке задано в заголовку відповіді {{httpheader("Attribution-Reporting-Register-Source")}}), коли користувач клацає посилання. Дивіться подробиці в [API звітування про атрибуцію](/uk/docs/Web/API/Attribution_Reporting_API).
+
+    Є дві версії цього атрибуту, які можна задати:
```
Comment (undead404) [inline files/uk/web/html/element/a/index.md:26]:
```suggestion
    Можна задати одну з двох версій цього атрибуту:
```

File: files/uk/web/html/element/a/index.md [hunk 5]
```diff
@@ -17,6 +17,28 @@ browser-compat: html.elements.a
...
+
+    - Булева версія, тобто просто запис `attributionsrc`. Це задає те, що ви хочете, аби до того ж сервера, що й заданий в атрибуті `href`, надсилався заголовок {{httpheader("Attribution-Reporting-Eligible")}}. Це прийнятно, коли реєстрація джерела атрибуції відбувається на тому ж самому сервері.
+    - Значення, що вміщає один чи більше URL, наприклад:
```
Comment (undead404) [inline files/uk/web/html/element/a/index.md:29]:
```suggestion
    - Значення, що вміщає один або кілька URL, наприклад:
```

File: files/uk/web/html/element/a/index.md [hunk 6]
```diff
@@ -17,6 +17,28 @@ browser-compat: html.elements.a
...
+
+      ```html
+      attributionsrc="https://a.example/register-source
+      https://b.example/register-source"
+      ```
+
+      Це корисно в тих випадках, коли запитаний ресурс не перебуває на тому сервері, який ви контролюєте, або просто треба обробляти реєстрацію джерела атрибуції на іншому сервері. У такому випадку можна задати один або більше URL як значення `attributionsrc`. Коли стається запит ресурсу, на URL, задані в `attributionsrc`, надсилається заголовок {{httpheader("Attribution-Reporting-Eligible")}}, на додачу до походження ресурсу. Ці URL можуть потім відповісти заголовком {{httpheader("Attribution-Reporting-Register-Source")}}, щоб завершити реєстрацію.
```
Comment (undead404) [inline files/uk/web/html/element/a/index.md:36]:
```suggestion
      Це корисно в тих випадках, коли запитаний ресурс не перебуває на сервері під вашим контролем або просто хочеться обробляти реєстрацію джерела атрибуції на іншому сервері. Тоді можна задати одну або більше адрес URL як значення `attributionsrc`. Коли відбувається запит на ресурс, заголовок {{httpheader("Attribution-Reporting-Eligible")}} надсилається на адреси URL, задані в `attributionsrc `, а не лише за походженням ресурсу. Ці адреси URL потім можуть відповісти заголовком {{httpheader("Attribution-Reporting-Register-Source")}} відповідно для завершення реєстрації.
```

File: files/uk/web/html/element/a/index.md [hunk 7]
```diff
@@ -17,6 +17,28 @@ browser-compat: html.elements.a
...
+
+      > [!NOTE]
+      > Задання кількох URL означає те, що на одну можливість може бути зареєстровано більш ніж одне джерело атрибуції. Можна, наприклад, мати різні кампанії, успішність яких вимірюється, що включає генерування різних звітів щодо різних даних.
```
Comment (undead404) [inline files/uk/web/html/element/a/index.md:39]:
```suggestion
      > Задання кількох адрес URL означає те, що на один елемент можна зареєструвати кілька джерел атрибуції. Можна, наприклад, мати різні кампанії, для вимірювання успішності яких потрібно генерувати різні звіти щодо різних даних.
```

File: files/uk/web/html/element/a/index.md (hunk 2 above)
Comment (alinkedd) [reply files/uk/web/html/element/a/index.md]: апрув, я тобі довіряю :)

**Review by alinkedd — APPROVED** [review]

**Comment by undead404** [comment]
Дякую!

**Pull request merged**